v3/src/models/tiles/tile-component-info.ts

```diff
@@ -27,7 +34,12 @@ export function registerTileComponentInfo(tileComponentInfo: ITileComponentInfo)
 
 // Tool id, e.g. kDrawingTileType, kGeometryTileType, etc.
 // undefined is supported so callers do not need to check the id before passing it in
+// toLowerCase() for legacy support of tool names
 export function getTileComponentInfo(type?: string) {
-  // toLowerCase() for legacy support of tool names
   return type ? gTileComponentInfoMap.get(type.toLowerCase()) : undefined
 }
+
+export function getTileComponentIcon(type?: string) {
+  const cIcon = type && gTileComponentInfoMap.get(type.toLowerCase())?.Icon
+  return cIcon
```

Comment:
```suggestion
  return getTileComponentInfo(type)?.Icon
```
and then you can put the `toLowerCase()` comment back where it was above.